apps/microtvm-vm/base-box/packer.json.template

```diff
@@ -0,0 +1,43 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+{
```

Comment:
Please link to a schema at the top level or something that is a reference to this json schema .
What do these keys mean  ? How much of an expert should a uTVM developer be to configure this or use it as is ?

Reply:
i've moved this into a new standardized release tool. uTVM developers are not expected to ever need to configure this. someone maintaining e.g. the Zephyr integration or another RTOS/platform integration may need to understand this, but the packer docs are [extensive](https://www.packer.io/docs).